README.md

```diff
@@ -70,6 +70,7 @@ compressFormat | Can be either JPEG, PNG or WEBP (android only).
 quality | A number between 0 and 100. Used for the JPEG compression.
 rotation | Rotation to apply to the image, in degrees, for android. On iOS, rotation is limited (and rounded) to multiples of 90 degrees.
 outputPath | The resized image path. If null, resized image will be stored in cache folder. To set outputPath make sure to add option for rotation too (if no rotation is needed, just set it to 0).
+outputImageName | The new name of image redized. (if no need outputPath, just set to null). Android only.
```

Comment:
there a small typo ;) Also I'm not sure about what you mean by  "if no need outputPath, just set to null"?
I think something like `outputFilename | The name of the output file (if not defined, a random name will be generated). Android only.`

Reply:
About the text doc think about the following situation:

with outputPath:
ImageResizer.createResizedImage(uri, width, height, format, quality, rotation, outputPath, outputFileName )

without outputPath
ImageResizer.createResizedImage(uri, width, height, format, quality, rotation, null, outputFileName )

But your text got more objective, I'm going to change